scripts/publish-workflow.sh

```diff
@@ -0,0 +1,7 @@
+CircleCIToken=$1
```

Comment:
@sfsholden - Any reason for not creating a script/approve-workflow as well? You already have the slack notification so you have the workflow id?

I _think_ you could also do https://circleci.com/docs/api/v2/#approve-a-job?

Reply:
@vazexqi I personally think it might be better to have an official "approve" process in circleci for security reasons. This would help prevent instances where someone may have accidentally launched the script without intending to publish. It also leaves a history of that individual who approved the workflow.

I'll send you a copy of a demo using my forked branch. Feel free to let me know what you think of the flow